src/apps/chifra/pkg/types/raw.go

```diff
@@ -0,0 +1,30 @@
+package types
```

Comment:
I really like having a collection of "FromTheNode" and "FromTrueBlocks" data structures. Gives us a lot of flexibility. Maybe add a comment here?